src/Physics/Multinucleon/XSection/MECScaleVsW.cxx

```diff
@@ -0,0 +1,166 @@
+//_________________________________________________________________________
+/*
+ Copyright (c) 2003-2020, The GENIE Collaboration
+ For the full text of the license visit http://copyright.genie-mc.org
+
+ Original code contributed by J.Tena and M.Roda
+ Substantial code refactorizations by the core GENIE group.
+*/
+//_________________________________________________________________________
+
+#include "Framework/Algorithm/AlgConfigPool.h"
+#include "Framework/Messenger/Messenger.h"
+#include "Physics/Multinucleon/XSection/MECScaleVsW.h"
+#include "Framework/Utils/StringUtils.h" 
+#include "Framework/ParticleData/PDGCodes.h"
+#include "Framework/ParticleData/PDGLibrary.h"
+
+using namespace genie;
+
+//_________________________________________________________________________
+MECScaleVsW::MECScaleVsW() : 
+  XSecScaleMap()
+{
+  
+}
+//_________________________________________________________________________
+MECScaleVsW::MECScaleVsW(string config) : 
+  XSecScaleMap(config) 
+{
+  
+}
+//_________________________________________________________________________
+MECScaleVsW::~MECScaleVsW()
+{
+
+}
+//_________________________________________________________________________
+void MECScaleVsW::Configure(const Registry & config)
+{
+    Algorithm::Configure(config);
+    this->LoadConfig();
+}
+//____________________________________________________________________________
+void MECScaleVsW::Configure(string config)
+{
+    Algorithm::Configure(config);
+    this->LoadConfig();
+}
+//_________________________________________________________________________
+double MECScaleVsW::GetScaling( const Interaction & interaction ) const
+{
+  double Q0 = interaction.Kine().GetKV(kKVQ0) ; 
+  double Q3 = interaction.Kine().GetKV(kKVQ3) ; 
+
+  return GetScaling( Q0, Q3 ) ; 
+}
+//_________________________________________________________________________
+double MECScaleVsW::GetScaling( const double Q0, const double Q3 ) const
+{
+  // Get the vectors that include the kinematic limits of W for a given event:
+  std::vector<double> W_limits, weights ; 
+  GetVectorWithLimits( W_limits, weights,Q0, Q3, fDefaultWeight ) ;
+
+  // The Scaling is done using the "experimenter's W", which assumes a single nucleon
+  // See motivation in : https://arxiv.org/pdf/1601.02038.pdf
+  // Calculate event W:
+  double Mn = ( PDGLibrary::Instance()->Find(kPdgProton)->Mass() + PDGLibrary::Instance()->Find(kPdgNeutron)->Mass() ) * 0.5 ;  // Nucleon mass
```

Comment:
This is where the varaible is defined already. but it is deprecated. 

https://github.com/GENIE-MC/Generator/blob/68956cac1aa1546e4e12c0143189f35317331d55/src/Framework/Conventions/Constants.h#L77

Reply:
make it `static`